src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -65,6 +68,7 @@
 
     public static final int STS_CREDENTIALS_DURATION_SECONDS = 3600;
     private final String accessKey;
+    private String awsRegion;
```

Comment:
If it's required below, it should have a default value, the best one being the previously hardcoded one.

Reply:
added the default value